graph/graph.py

```diff
@@ -15,7 +15,7 @@
     import networkx as nx
     from networkx.exception import NetworkXNoPath, NodeNotFound
 except ImportError:
-    PACKAGE = "networkx==2.5.1"
+    PACKAGE = "networkx==2.8.6"
```

Comment:
Aldo, could you nuke this try except `ImportError` import `networkx` here? I mean, I appreciated that you also took the opportunity to fix this part, but this try import here was the approach we used to use many years ago (if you look on `git blame` I added this back in 2019, time flies) when a NApp actually didn't use to install its dependencies (`install_requires=read_requirements()` on setup.py). So, you can nuke this entire try except here.